ext/mbstring/tests/mb_ereg1.phpt

```diff
@@ -19,15 +19,16 @@ foreach ($a as $args) {
 }
 ?>
 ===DONE===
---EXPECTF--	
```

Comment:
I'd prefer to avoid whitespace changes in bug fixes (and feature implementations, for that matter), because that makes it easier to read the diff. (Yeah, I know that these trailing whitespaces shouldn't be there in the first place, but a separate commit/pull request might be more appropriate to fix this.)